### ISSUE TITLE
raise button where I could only call

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#855]
Get rid of the incorrect raise button that can occur in a limit game with a short-all in raise.

```
Dealing #242998: 4000 8000 Archie

🚂 Chuck blinds 2000
deadhead blinds 4000
2     pokerchimp    9275    0  [  ] [  ] [  ] [  ] [  ]
3     smalltalkdan  74525    0  [  ] [  ] [  ] [  ] [  ]
5     🚂 Chuck       2875 2000  [  ] [  ] [  ] [  ] [  ]
6     deadhead      16825 4000  [  ] [  ] [  ] [  ] [  ]
7   > jrx           50500    0  [  ] [  ] [  ] [  ] [  ]
My cards are: [3h] [Kh] [5s] [Kd] [9s]
jrx folds
🎴NNR13: nh dan, 
pokerchimp calls
smalltalkdan: tx
smalltalkdan calls
🚂 Chuck calls
It is 0 to you
deadhead raises 4000 to 8000
pokerchimp raises 1275 to 9275 and is all-in
smalltalkdan calls
🚂 Chuck calls and is all-in
It is 1275 to you
```
At this point I had a raise button, but since pokerchimp's raise was a short one, I shouldn't have had the ability to raise. I clicked the raise button and it was treated as a call, so it wasn't game-endingly horrible, but it was distracting (since I then had to open this issue).
```
deadhead calls
2     pokerchimp    0 9275  [  ] [  ] [  ] [  ] [  ]
3     smalltalkdan  65250 9275  [  ] [  ] [  ] [  ] [  ]
5   > 🚂 Chuck       0 4875  [  ] [  ] [  ] [  ] [  ]
6     deadhead      11550 9275  [  ] [  ] [  ] [  ] [  ]
7    [jrx]          50500[   0] 
My cards are: [3h] [Kh] [5s] [Kd] [9s]
🚂 Chuck draws 1
deadhead draws 3
My cards are: [Td] [Kh] [8c] [Kd] [2s]
pokerchimp draws 2
smalltalkdan draws 3
FinishedDrawing not yet implemented
It is 0 to you
deadhead: GUI gave me a raise button when it shouldn't have
```

**Comment by ctm** [comment]
I just got around to looking at this now and it's a bit embarrassing in that it's simply a piece of functionality that I didn't add to action tracker when it was expanded to accommodate raise buttons. There's a comment where the work needs to be done. The funny thing is that this is actually a simpler piece of functionality than what we have to do for stud, because stud has to worry about the concept of a completion.

So, this particular issue has been around since I added the graphical buttons, but it only comes into play when there's all-in action that isn't sufficient to re-open betting but there is another caller who still has chips.
